src/commands/switch/organisation.ts

```diff
@@ -0,0 +1,71 @@
+import { RequestBase } from '../../utils/request-base';
+import chalk from 'chalk';
+import ErrorResponse from '../../error-response';
+import { ICommand, IOrganisation, ISwitchOrganisationQuestionAnswers } from '../../types';
+import { CommandBuilder } from 'yargs';
+import { getProperty, processAnswers, updateLineSalableRc } from '../../utils';
+import { SWITCH_ORGANISATION_QUESTIONS } from '../../questions';
+
+const builder: CommandBuilder = {};
+
+const handler = async () => {
+  try {
+    const organisations = await RequestBase<IOrganisation[]>({
+      method: 'GET',
+      endpoint: `organisations`,
+    });
+
+    if (!organisations) {
+      // eslint-disable-next-line no-console
+      console.log(chalk.yellow(`No organisations found`));
+      return;
+    }
+
+    const orgNames = organisations?.map((org) => org.name);
+
+    const { organisation } = await processAnswers<ISwitchOrganisationQuestionAnswers>(
+      SWITCH_ORGANISATION_QUESTIONS.ORGANISATION(orgNames)
+    );
+
+    const targetOrg = organisations.find((org) => org.name === organisation);
+
+    if (!targetOrg) {
+      // eslint-disable-next-line no-console
+      console.error(chalk.red('Specified organisation cannot be found. Please try again.'));
+      return;
+    }
+
+    const currentToken = await getProperty('ACCESS_TOKEN');
+
+    const newSessionToken = await RequestBase<string>({
+      method: 'POST',
+      endpoint: `cli/switch-organisation`,
+      body: {
+        organisation: targetOrg?.name || '',
```

Comment:
Should be able to drop this because of the check on `L32`? Is there any reason an org would exist but not have a name? I don't think so, right? 🤔